bindings/npm/README.md

```diff
@@ -17,18 +17,19 @@ cd bindings/npm
 npm install
 ```
 
+## Usage
+
+To try the generated library, you can use:
+- [node.js demo](/demo/nodejs)
+
 ### In docker
 
 ```bash
-docker build -f bindings/npmnpm/Dockerfile -t rgb-sdk-npm .
-docker run --rm -v $(pwd):/opt/mount --entrypoint bash \
-    rgb-sdk-npm \
-    -c 'mkdir -p /opt/mount/librgb/target/release /opt/mount/bindings/npm/build/Release \
-    && cp /rgb-sdk/target/release/librgb.so /opt/mount/librgb/target/release/librgb.so \
-    && cp /rgb-sdk/rgb.node /opt/mount/bindings/npm/build/Release/rgb.node'
-```
+$ docker build -f bindings/npm/Dockerfile -t rgb-sdk-npm .
```

Comment:
This will make the thing unusable with anything else than running a sample demo right inside the docker container

Reply:
I wasn't sure why a volume was added and copied the `rgb.node` and `librgb.so` to there. I modified the readme to add instruction to run the demo. Earlier it wasn't clear to me what the docker file is supposed to do? And in the readme instruction, we were just building it and copying those two files into a volume.

It's possible that I am missing the purpose of the docker build. I just made it to run the `example.js`. If it's not what it was ment for, do suggest and I will change accordingly.

Reply:
The docker is used to build `rgb.node`, which is a compiled NodeJS module file, which must be used on the local machine. That is the purpose of the docker and it has nothing to do with the example

Reply:
Ok so I will remove the demo part. Do you want those copying instructions into a volume to be kept?

Reply:
Yes, they have to be kept. There was nothing to change in this file other than directory names; no lines should be added or removed